charts/quickwit/values.yaml

```diff
@@ -50,6 +53,9 @@ searcher:
   # Extra env for searcher
   extraEnv: {}
     # KEY: VALUE
+  extraEnvFrom: {}
+    # - secretRef:
+    #     name: quickwit-searcher
```

Comment:
Here we can add:

```yaml
     # - configMapRef:
               name: configmap1
```

Reply:
Thanks for your example, I was overcomplicating it :joy: